test/show_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/character'
+require './lib/show'
+
+class ShowTest < Minitest::Test
+
+  def setup
+    @kitt = Character.new({name: "KITT", actor: "William Daniels", salary: 1_000_000})
+    @michael_knight = Character.new({name: "Michael Knight", actor: "David Hasselhoff", salary: 1_600_000})
+    @knight_rider = Show.new("Knight Rider", "Glen Larson", [@michael_knight, @kitt])
+  end
+
+  def test_it_exists
+    assert_instance_of Show, @knight_rider
+  end
+
+  def test_show_attributes
+    assert_equal "Knight Rider", @knight_rider.name
+    assert_equal "Glen Larson", @knight_rider.creator
+    assert_equal [@michael_knight, @kitt], @knight_rider.characters
+    assert_equal 2600000, @knight_rider.total_salary
+  end
```

Comment:
good testing here. Generally the `attributes` test we'd write are only going to check that initialized values are set as we'd expect. The `total_salary` method should be in its own test.